docs/pages/enroll-resources/database-access/enroll-aws-databases/rds-oracle.mdx

```diff
@@ -393,6 +393,8 @@ Other clients can use:
 
 This method also enables use of various graphical clients, as explained in [Oracle graphical clients](../../../connect-your-client/gui-clients.mdx#oracle-graphical-clients) section.
 
+(!docs/pages/includes/database-access/oracle-multihost.mdx!)
```

Comment:
does this apply to RDS?

Reply:
Yes, but rethinking this, it does feel very uncommon. Since we will still have reference for documenting this option I think we can remove this part.